src/styles/header.scss

```diff
@@ -0,0 +1,151 @@
+@media screen and (max-width: 1024px) {
+  .header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    font-size: 12px;
+    font-weight: 500;
+
+    text-transform: uppercase;
+
+    padding: 0 50px;
+
+    box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+
+    &__logo {
+      display: flex;
+      align-items: center;
+    }
+
+    &__menu {
+      display: flex;
+      list-style-type: none;
+      margin: 0;
+      padding: 0;
+    }
+
+    &__menu-item {
+      margin-right: 20px;
+
+      &:first-child {
+        margin-left: 0;
+      }
+
+      &:last-child {
+        margin-right: 0;
+      }
+    }
+
+    &__menu-link {
+      text-decoration: none;
+      text-transform: uppercase;
+      display: flex;
+      align-items: center;
+      height: 60px;
+      position: relative;
+
+      color: $main-accent-color;
+
+      transition: color 300ms ease;
+
+      &:hover {
+        color: $blue-accent-color;
+      }
+
+      &--active {
+        position: relative;
+        color: $blue-accent-color;
+
+        &::after {
+          content: '';
+          display: block;
+          width: 100%;
+          height: 4px;
+          background-color: $blue-accent-color;
+          position: absolute;
+          bottom: 0;
+          border-radius: 8px;
+
+          transition: background-color 300ms ease;
+        }
+      }
```

Comment:
In the media query for max-width: 1024px, the active link class is defined as `&--active`. However, in the media query for min-width: 1024px, it is defined as `&.is-active`. Ensure consistency in class naming to avoid confusion and potential styling issues.